common/src/main/java/org/jetbrains/annotations/ApiStatus.java

```diff
@@ -72,6 +72,23 @@ private ApiStatus() {
   })
   public @interface Internal {}
 
+  /**
+   * <p>Indicates that a public API of the annotated element (class, method or field) is subject to deprecation in a future version.
```

Comment:
I would write that the annotated element isn't supposed to be used in new code because a better alternative exists, but it's ok to have its usages in existing code (and therefore its usages aren't highlighted with warnings).